scripts/userstyles.yml

```
@@ -531,6 +534,13 @@ userstyles:
     readme:
       app-link: "https://jisho.org"
     current-maintainers: [*Lichthagel]
+  keybr.com:
+    name: keybr.com
+    categories: [productivity]
+    color: mauve
```

```suggestion
    color: text
```
Arguably no accent/brand color for keybr.com so just text will do.